packages/core/content-manager/admin/src/components/ConfigurationForm/Fields.tsx

```diff
@@ -47,17 +104,24 @@ interface FieldsProps extends Pick<EditLayout, 'metadatas'>, Pick<FieldProps, 'c
 
 const Fields = ({ attributes, fieldSizes, components, metadatas = {} }: FieldsProps) => {
   const { formatMessage } = useIntl();
+  const [activeField, setActiveField] = React.useState<DragAndDropField | null>(null);
 
   const layout = useForm<ConfigurationFormData['layout']>(
     'Fields',
     (state) => state.values.layout ?? []
   );
+  const containers = layout.reduce(
+    (acc, row, index) => ({
+      ...acc,
+      [`container-${index}`]: row,
+    }),
+    {} as Record<string, ConfigurationFormData['layout'][number]>
+  );
```

Comment:
This is probably unnecessary, in the beginning I just found it easier to work with an object. I'll look at removing and relying on the layout array directly.